evansmusomi/react-native-102
# Adds Android Splash Screen

*What this does*
- Adds a Splash Screen for Android

*How to verify*
- When you launch the app, you should see a splash screen with the name of the app before the auth screen is displayed
